LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -688,6 +692,58 @@ function showJSONendpoint() {
 	dialog_message.show();
 }
 
+/**
+ * Purpose: Create New Row Requests
+ * @param catID - Form ID passing in for new request
+ * @return - Creates new request inline on grid
+ */
+function createRequest(catID) {
+    const portalAPI = LEAFRequestPortalAPI();
+    portalAPI.setBaseURL('./api/?a=');
+    portalAPI.setCSRFToken(CSRFToken);
+    dialog.setTitle('New Request Title');
+    dialog.setContent('<div><h4>Title of Request</h4>'
+                    + '<p>Enter a title and save changes to create new requests.<br/>They will appear at the top of the table.</p>'
+                    + '<span class="text">'
+                    + '<input id="newTitle" type="text" name="newTitle" required placeholder="Request Title" style="font-size: 1.3em; font-family: monospace"/>'
+                    + '</span>'
+                    + '</div>');
+    dialog.show();
+    let titleInput;
+    $('#button_save').on('click', function() {
```

Comment:
Couple things on this.  

titleInput needs verified.  The "if" will test true on " " and "0".  Neither of which I'm guessing are useful.

catID is unchecked before it gets used.  I know IE doesn't allow default values in function attributes, much less type-forcing.  This can be gotten around by modifying the "if" statement that uses it to something like this:  
if (typeof(Number(jj)) === "number" && Number(jj) > 0) {}

Reply:
Its ok if it tests true on numbers people tend to put only numbers, I don't know if theres only a space maybe some users do that? they like to have blank titles so they just put a space. I can do a check for that. And doing a check for catID I completely agree.